phac-nml/irida-next
# Display samples for workflow execution (ENHC0010030)

## What does this PR do and why?
_Describe in detail what your merge request does and why._

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/11295750/d6f6dcda-d0a3-43bc-8401-9ea886ec5db5)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Create a new project
2. Add a sample to the project and upload sequence files to the sample.
3. Go back to your new project samples page and select the new sample and launch a pipeline for it.
4. Go to workflow executions and select the new one.
5. Select the tab for `Samplesheet`
6. Files should be displayed as above.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
